### PR TITLE
chore: [IA-69] Remove title, update description and tests

### PR DESCRIPTION
**Pull request by pietro909**
## Short description
Remove the title and update the description.

## List of changes proposed in this pull request
- Move the title from the screen content to the header
- Replace the description with an infobox

## How to test
- to be able to observe the message, please set [the timeout in the dev server](https://github.com/pagopa/io-dev-api-server/blob/master/src/routers/payment.ts#L154) to something longer than 1s
- navigate under _Wallet_
- add a new card
- click on _save_
- see the new message in the webview

### After

Simulator Screen Shot - iPhone 11 - 2021-06-24 at 17 03 10

### Before

Screenshot 2021-06-14 at 12 06 56

### CODE REVIEW

**Comment by thisisjp** [comment]
@pietro909 looks like the font is wrong ;)

**Comment by pietro909** [comment]
Good catch @thisisjp ! 
[Here's](https://github.com/pagopa/io-app/pull/3169/files#diff-6bfcb9d77f5645ed22d61310cbcaae4100aebeb4f93d593ae5e717710f9d1495R51) the fix, and I've updated the screenshot in the description.

File: ts/components/wallet/PayWebViewModal.tsx [hunk 1]
```diff
@@ -169,10 +179,21 @@ export const PayWebViewModal = (props: Props) => {
...
+          testID={"PayWebViewModal-description"}
+        >
+          <InfoBox
+            iconName={"io-info"}
+            iconColor={IOColors.bluegreyDark}
+            iconSize={24}
+          >
+            <Text style={styles.description}>
+              {I18n.t("wallet.challenge3ds.description")}
+            </Text>
```
Comment (Undermaken) [inline ts/components/wallet/PayWebViewModal.tsx:193]:
```suggestion
<Label weight={"Regular"} color={"bluegrey"}>
   {I18n.t("wallet.challenge3ds.description")}
</Label>
```
`Label` is a typography component that ensures certain IO styles and features
Could it possibile to add it in place of using custom `Text`?

**Review by Undermaken — CHANGES_REQUESTED** [review]

File: ts/components/wallet/PayWebViewModal.tsx (hunk 1 above)
Comment (pietro909) [reply ts/components/wallet/PayWebViewModal.tsx:193]: Thank you @Undermaken for the suggestion! I missed that component 🤦 and I believe it is the right choice here.

That said, `Label` component [has fixed 16px size](https://github.com/pagopa/io-app/blob/master/ts/components/core/typography/Label.tsx#L13) whereas from the designs we need 14px. 
Adding a screenshot here for you and @thisisjp to evaluate it:

Screenshot 2021-06-25 at 09 30 55

Comment (thisisjp) [reply ts/components/wallet/PayWebViewModal.tsx:193]: Looks good to me, we'll review all the design library components later ;) thanks

Comment (pietro909) [reply ts/components/wallet/PayWebViewModal.tsx]: Updated https://github.com/pagopa/io-app/pull/3169/commits/82a268155eeb5e801ab9ceb93a1d6fc96be9e5f9

**Review by Undermaken — APPROVED** [review]

**Pull request merged**